### ISSUE TITLE
m.inven.co.kr

### ISSUE THREAD

**Issue by swbaek81** [yous/YousList#308]
광고 제거 부탁드립니다 감사합니다

아이폰 사파리 애드가드프로
youslist

https://m.inven.co.kr/board/webzine/2097/2153885

![image](https://github.com/yous/YousList/assets/15683789/27bb5df3-b8a3-4181-acf7-73b6a3e63896)

![image](https://github.com/yous/YousList/assets/15683789/43efdc87-d0fe-47b3-ba04-898f83f06e88)

**Comment by yous** [comment]
네, microShield에 등록되어 있네요. PC 환경에서는 차단되는데, iOS 환경에서는 새로고침 하다보면 보이네요.
